roles/epel/tasks/main.yml

```diff
@@ -0,0 +1,19 @@
+---
+- name: Check if EPEL is installed
+  stat:
+    path: /etc/yum.repos.d/epel.repo
+  register: st
+  when: ansible_distribution == "RedHat"
+
+- name: Download EPEL RPM
+  get_url:
+    url: "https://dl.fedoraproject.org/pub/epel/epel-release-latest-{{ ansible_distribution_major_version }}.noarch.rpm"
+    dest: "/tmp/epel{{ ansible_distribution_major_version }}.rpm"
+    mode: 0440
+  when: not st.stat.exists and ansible_distribution == "RedHat"
+
+- name: Install EPEL repo
+  yum:
+    name: "/tmp/epel{{ ansible_distribution_major_version }}.rpm"
```

Comment:
Temp file is not needed, `yum` can download URL directly, see: http://docs.ansible.com/ansible/yum_module.html

> You can also pass a url or a local path to a rpm file